src/components/generics/icons/IconYeti.vue

```diff
@@ -0,0 +1,11 @@
+<template>
+  <fa-layers>
+    <fa-icon class="icon-yeti" icon="exclamation" transform="shrink-6 rotate-45" mask="square" />
+  </fa-layers>
+</template>
+
+<style>
```

Comment:
Always use scoped styles, otherwise, your class will be valid for entire project 

```suggestion
<style scoped>
```